api/deleteUser.js

```diff
@@ -0,0 +1 @@
+async () => ({ result: 'delete' });
```

Comment:
Short and clear
![think](https://user-images.githubusercontent.com/4405297/120502139-d936f600-c3ca-11eb-8783-6ba949085545.gif)